app/views/content/non-uk-teachers/visas-for-non-uk-trainees.md

```diff
@@ -106,87 +125,45 @@ Non-UK salaried trainee teachers must earn a minimum salary to be eligible for a
 
 ### Dependants
 
-You will not need to be sponsored for a Student or Skilled Worker visa if you have permission to live in the UK as a dependant (usually partner) of someone on a visa. You can work or study without a sponsor for the duration of your partner’s visa. 
-
+You will not need to be sponsored for a Student or Skilled Worker visa if you have permission to live in the UK as a dependant of someone on a visa, usually a partner. You can work or study without a sponsor for the duration of your partner’s visa. 
 
+You may be able to bring your partner and children with you if you are on a Skilled Worker visa. 
 
-## Visa fees and other charges
-
-You'll usually need to show you have enough money to support yourself in England by providing proof of savings or income. You’ll be guided through this process when you apply for your visa.
 
 ### Visa fees
 
-You’ll also have to pay [visa fees](https://www.gov.uk/visa-fees).
-
-### Immigration Health Surcharge
+You will need to pay visa fees. You can work out the cost of your UK visa application by [using the visa fees tool](https://www.gov.uk/visa-fees).  
 
-You may need to [pay for UK healthcare](https://www.gov.uk/healthcare-immigration-application) as part of your immigration application – this is called the Immigration Health Surcharge (IHS).
+When you apply for a visa, you will usually need to show you have enough money to support yourself in England by providing proof of savings or income.  
 
-However, some students can claim a refund of the IHS if they:
+### Immigration Health Surcharge
 
-* are a full-time student in UK higher education
-* have a visa starting on or after 1 January 2021
-* have a European Health Insurance Card (EHIC) issued in an EU country
-* do not work in the UK
+You may need to [pay for UK healthcare](https://www.gov.uk/healthcare-immigration-application) as part of your immigration application. This is called the Immigration Health Surcharge (IHS).
 
-Learn more about [getting an IHS refund](https://www.gov.uk/guidance/immigration-health-surcharge-for-eu-students-in-the-uk).
+However, [some students can claim a refund of the IHS](https://www.gov.uk/guidance/immigration-health-surcharge-for-eu-students-in-the-uk) depending on their circumstances.
 
-## Contact
 
-If you have general questions about the information on this page, you can email us at teach.inengland@education.gov.uk.
+## Get help
 
-You can also [contact UK Visas and Immigration for help](https://www.gov.uk/contact-ukvi-inside-outside-uk).
+[Contact UK Visas and Immigration for help](https://www.gov.uk/contact-ukvi-inside-outside-uk).
 
-You may be eligible for personalised support from a [teacher training adviser](/teacher-training-advisers). They can help with your application to a training provider or school.
+You may be eligible for personalised support from a [teacher training adviser](/teacher-training-advisers). They can help with your teacher training application.
```

Comment:
I wonder if we should take this link out - we want people to ring/chat to the contact centre. If we send them to the TTA page, might they just fill in the form on there?